### DISCUSSION TITLE
mrsk website and docs

### DISCUSSION THREAD

**Discussion by Arvind644** [basecamp/kamal#238] [Ideas]
Hii, I recently worked with mrsk and I really liked it a lot. I want to work on creating a website for mrsk and also to contribute in setting up documentation for mrsk using either hugo or docasaurus. And I want to contribute to docs. Please let me know if it sounds good.

**Comment by nickhammond** [comment]
@Arvind644 Looks like a new site was just published https://github.com/mrsked/mrsk-site
